app/packs/src/apps/mydb/elements/details/formats/FormatComponent.js

```diff
@@ -103,10 +103,10 @@ FormatComponentHeader.propTypes = {
 };
 
 const FormatComponent = ({
-  list, variant, onSave, onFormat, onClose
+  list, isSaved, onSave, onFormat, onClose
```

Comment:
The `isSaved` prop does not seem to be used.

Reply:
Correct, but it should be. I implemented this completely now.